src/test/crimson/seastore/CMakeLists.txt

```diff
@@ -38,6 +38,18 @@ target_link_libraries(
   crimson::gtest
   crimson-seastore)
 
+add_executable(unittest-object-data-handler
+  test_object_data_handler.cc
+  ../gtest_seastar.cc)
+add_ceph_unittest(unittest-object-data-handler
+  --memory 256M --smp 1)
```

Comment:
@athanatos 
```diff
diff --git a/src/test/crimson/seastore/CMakeLists.txt b/src/test/crimson/seastore/CMakeLists.txt
index a3a88e67bc3..f1585bf0ec0 100644
--- a/src/test/crimson/seastore/CMakeLists.txt
+++ b/src/test/crimson/seastore/CMakeLists.txt
@@ -42,7 +42,7 @@ add_executable(unittest-object-data-handler
   test_object_data_handler.cc
   ../gtest_seastar.cc)
 add_ceph_unittest(unittest-object-data-handler
-  unittest-object-data-handler --memory 256M --smp 1)
+  --memory 256M --smp 1)
 target_link_libraries(
   unittest-object-data-handler
   crimson::gtest
```

Reply:
otherwise the command would be

```sh
224: Test command: /home/jenkins-build/build/workspace/ceph/build/bin/unittest-object-data-handler "unittest-object-data-handler" "--memory" "256M" "--smp" "1"
```